Registry/Registry.EM_COMMON

```diff
@@ -893,6 +893,13 @@ state    real   T2OBS            ij     misc        1         -      r        "T
 state    real   Q2OBS            ij     misc        1         -      r        "Q2OBS"        "2-m mixing ratio from analysis "   "kg/kg"
 state    real   IMPERV           ij     misc        1         -      i012rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)      "IMPERV"       "Impervious surface fraction NLCD"  "percent"
 state    real   CANFRA           ij     misc        1         -      i012rhd=(interp_mask_field:lu_index,iswater)u=(copy_fcnm)      "CANFRA"       "Satellite canopy fraction"         "percent"
+state    real   LAI_PX           ij     misc        1         -      rh        "LAI_PX"       "LAI used for PX LSM"              "m2/m2"
```

Comment:
@coastwx 
I know that these are only 2d arrays, but can we package these? We can base it on pxlsm_modis_veg alone, but that would mean adding in a test in share/module_check_a_mundo.F to make sure that the PX scheme is active if pslsm_modis_veg =1.